src/components/input.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from "react"
+import styled from "styled-components"
+import colors from "../assets/colors"
+import * as fonts from "../assets/fonts"
+import { block } from "./helpers"
+
+interface InputProps extends React.HTMLProps<HTMLInputElement> {
```

Comment:
Yeah I was thinking the same thing, didn't want to prematurely extract common styles but I think it makes sense in this case. I'll create a mixin for this.